drivers/io-optimize.h

```diff
@@ -42,7 +42,10 @@ struct opio_list {
 	struct opio        *tail;
 };
 
+#define UIO_FASTIOV 8
```

Comment:
Is this a hardcoded value in the kernel or could it be configured? i.e. should we/could we be reading this from the kernel?

Reply:
It is present in `linux/uio.h`, but if we include that we get a double definition of `struct iovec` between `linux/uio.h` and `bits/uio.h` included through `fcntl.h`